shared-libraries/lib/utils/exchangeReq.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.getExchangeRates = undefined;
+
+var _rates = require('shared-libraries/lib/fixtures/rates');
```

Comment:
Why require? Why not import? Let's try to remain consistent. Is there a reason for this?